src/components/table/makeSelectable.js

```diff
@@ -612,8 +608,27 @@ function makeSelectable(BaseTable) {
 
         isTargetSlider = event => event.target?.role === 'slider';
 
-        // Workaround for focus conflicting with Blueprint components in Search Quick Filters, not needed once Blueprint table is integrated
-        isQuickFilter = event => event.target?.dataset && 'radixCollectionItem' in event.target.dataset;
+        // Workaround for focus conflicting with Blueprint components for QuickSearch result, recent items and Quick Filters
+        isTargetQuickSearch = event => {
+            if (!event.target) {
+                return false;
+            }
+
+            const { className, dataset } = event.target;
+
+            if (
```

Comment:
🔧 might be worth adding comments over each of these `if` blocks detailing exactly what its targeting. These class names might be a easy enough to put together with the comment overtop the function but the dataset strategy below becomes much harder

Reply:
Something like this

```js
        if (!event.target) {
            return false;
        }

        const { className, dataset } = event.target;

        // QuickSearch Recent Item
        if (className?.includes('quickSearchRecentItem')) {
            return true;
        }

        // Quick Search Result Item
        if (className?.includes('quickSearchResultItem')) {
            return true;
        }

        // Blueprint's <FilterChip>
        if (dataset && ('radixCollectionItem' in dataset)) {
            return true;
        }

        // Blueprint's <SmallList>
        if (dataset && 'bpSmallListItem' in dataset) {
            return true;
        }

        return false;
```